confluent_kafka/avro/cached_schema_registry_client.py

```diff
@@ -21,15 +21,23 @@
 #
 import json
 import logging
+import warnings
 from collections import defaultdict
 
-import requests
+from requests import Session, utils
 
 from .error import ClientError
 from . import loads
 
+# Python 2 considers int an instance of str
+try:
+    string_types = basestring  # noqa
+except NameError:
+    string_types = str,
```

Comment:
I think it is better to make this tuple construct explicit so the trailing comma doesn't come off as a mistake

Reply:
That's not actually intended to be a tuple. This should be singular as well.